test/aircraft/Pilot/Pilot.spec.js

```diff
@@ -409,37 +407,53 @@ ava('.climbViaSID() correctly configures MCP and returns correct response when a
     t.true(pilot._mcp.altitude === 11000);
 });
 
-ava('.conductInstrumentApproach() returns error when no runway is provided', (t) => {
+ava('.conductInstrumentApproach() returns failure message when no runway is provided', (t) => {
+    const aircraftModel = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK, createNavigationLibraryFixture());
     const expectedResult = [false, 'the specified runway does not exist'];
-    const pilot = createPilotFixture();
-    const result = pilot.conductInstrumentApproach(approachTypeMock, null);
+    const result = aircraftModel.pilot.conductInstrumentApproach(aircraftModel, approachTypeMock, null);
+
+    t.deepEqual(result, expectedResult);
+});
+
+ava('.conductInstrumentApproach() returns failure message when assigned altitude is lower than minimum glideslope intercept altitude', (t) => {
+    const aircraftModel = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK, createNavigationLibraryFixture());
+
+    aircraftModel.mcp.setAltitudeFieldValue(1);
+
+    const expectedResult = [false, {
```

Comment:
set this above `aircraftModel` assignment

Reply:
done